lib/Sema/ConstraintSystem.h

```diff
@@ -2261,14 +2261,27 @@ class ConstraintSystem {
     return typeVar->getImpl().getRepresentative(getSavedBindings());
   }
 
-  /// Gets the a VarDecl, and the type of its attached property wrapper if the
-  /// resolved overload has an attached property wrapper.
-  ///
-  /// \return A pair of the VarDecl and the attached property wrapper type if
-  ///         the given resolved overload has an attached property wrapper.
+  /// Gets the VarDecl associateed with resolvedOverload, and the type of the
+  /// storage wrapper if the decl has an associated storage wrapper.
+  Optional<std::pair<VarDecl *, Type>>
+  getStorageWrapperInformation(ResolvedOverloadSetListItem *resolvedOverload) {
+    if (resolvedOverload->Choice.isDecl()) {
```

Comment:
Nit: Would be good to add an assert here to make sure that `resolvedOverload` points to something.